app/controllers/admin/users_controller.rb

```diff
@@ -1,11 +1,23 @@
 module Admin
   class UsersController < AdminController
 
-    def close_account
-      u = User.find(close_account_params)
-      u.close_account!
-      redirect_to user_search_path, notice: t('admins.user_search.account_closing_scheduled', name: u.username)
-    end
+  def close_account
```

Comment:
Please properly indent the code, it's a method in a class in a module, so four spaces of indentation here.